draft-jholland-quic-multicast.md

```diff
@@ -349,10 +352,15 @@ Note that the hash is on the encrypted packet to avoid leaking data about the en
 
 # Recovery {#recovery}
 
-TODO: Articulate key differences with {{RFC9002}}, mainly that the RTT for channel packets uses an estimated send time since the server doesn't necessarily know the exact timing of the send of the packet.
+TODO: Articulate key differences with {{RFC9002}}.
+The main known difference is that servers might not be running on the same devices that are sending the channel packets, therefore the RTT for channel packets might use an estimated send time that can vary according to the clock synchronization among servers and the deployment and implementation details of how the servers find out the sending timestamps of channel packets.
+Experience-based guidance on the recovery timing estimates is one anticipated outcome of experimenting with deployments of this experimental extension.
 
 All the new frames defined in this document except MC_ACK are ack-eliciting and are retransmitted until acknowledged to provide reliable, though possibly out of order, delivery.
 
+Note that recovery MAY be achieved either by retransmitting frame data that was lost and needs reliable transport either by sending the frame data on the unicast connection or by coordinating to cause an aggregated retransmission of widely dropped data on a multicast channel, at the server's discretion.
+However, the server in each connection is responsible for ensuring that any necessary server-to-client frame data lost by a multicast channel packet loss ultimately arrives at the client.
```

Comment:
Should there be a mention on difference between DGRAM and STREAM frames here? I think that might be a clearer separation between what is necessary data and what is not.

Reply:
Other kinds of frames besides streams also need to be retransmitted (like RESET_STREAM, as well as MC_INTEGRITY, MC_RETIRE, MC_LEAVE).  Likewise other kinds of frames besides DATAGRAM frames don't get retransmitted (like PING and PADDING).  I tried to leave it generic, but maybe we need a reference to https://www.rfc-editor.org/rfc/rfc9000.html#section-13.3 and https://www.rfc-editor.org/rfc/rfc9221.html#section-5.2 plus a similar section in this doc to list what gets retransmitted?